### PR TITLE
Delete unused Smarty block serialize

### PR DESCRIPTION
**Pull request by larssandergreen**
Overview
----------------------------------------
Not used in core or universe.

### CODE REVIEW

**Comment by larssandergreen** [comment]
Doesn't look related
```
Civi\Afform\AfformGetTest::testAfformAutocomplete
Undefined array key 0
```

Jenkins test this please

**Comment by colemanw** [comment]
`block.url.php` was only just recently added by @totten in #26861

**Comment by larssandergreen** [comment]
@colemanw Oh, funny. I just assumed it was old, how wrong I was.

File: CRM/Core/Smarty/plugins/block.serialize.php [hunk 1]
```diff
@@ -1,37 +0,0 @@
...
- * @param string $text
- *   {serialize} block contents from the template.
- * @param CRM_Core_Smarty $smarty
- *   The Smarty object.
- *
- * @return string
- *   the string, translated by gettext
- */
-function smarty_block_serialize($params, $text, &$smarty) {
-  return serialize($text);
```
Comment (colemanw) [inline CRM/Core/Smarty/plugins/block.serialize.php:36]: This looks like the sort of thing that might have been used by schema `tpl.sql` files.

Comment (larssandergreen) [reply CRM/Core/Smarty/plugins/block.serialize.php:36]: That would make sense. Can keep it if it is needed, but I can't find any trace of it.

**Comment by larssandergreen** [comment]
A couple of these same failures now:
```
Civi\Afform\AfformGetTest::testAfformAutocomplete
Undefined array key 0
```
Jenkins test this please

**Comment by larssandergreen** [comment]
Same again, I think maybe an issue with test-4.
Try once more, test this please.

**Pull request merged**